Tests/Functional/ClangModuleTests.swift

```diff
@@ -97,6 +97,13 @@ class TestClangModulesTestCase: XCTestCase {
             XCTAssertFileExists(prefix, ".build", "debug", "Baz")
         }
     }
+
+    func testDynamicLookup() {
```

Comment:
You're right, I renamed the test and added a comment. https://github.com/apple/swift-package-manager/pull/348